pkg/operator/controllers/genevalogging/genevalogging_controller.go

```diff
@@ -41,14 +42,17 @@ type Reconciler struct {
 }
 
 func NewReconciler(log *logrus.Entry, client client.Client, dh dynamichelper.Interface) *Reconciler {
-	return &Reconciler{
+	r := &Reconciler{
 		AROController: base.AROController{
-			Log:    log,
-			Client: client,
-			Name:   ControllerName,
+			Log:         log.WithField("controller", controllerName),
```

Comment:
If we pass the controller's name into the `AROController` instance and we expect every time a new `AROController` is instantiated, the logger must be updated with a `"controller"` field, perhaps we can move that into an `NewAROController()` constructor.